sale_commission_advance/models/account_move.py

```diff
@@ -0,0 +1,129 @@
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model
+    def create(self, vals):
+        rec = super(AccountMove, self).create(vals)
```

Comment:
And please add a docstring why you need to do this.

```suggestion
        rec = super().create(vals)
```